src/topics/follow.js

```diff
@@ -254,6 +253,9 @@ module.exports = function (Topics) {
 						}
 
 						if (data.userSettings.sendPostNotifications) {
+							if (parseInt(meta.config.removeEmailNotificationImages, 10) === 1) {
+								postData.content=postData.content.replace(/<img[^>]*>/, '');
```

Comment:
Should be `postData.content = postData.content.replace(/<img[^>]*>/, '');` to pass linter